templates/ContentGenerator/Instructor/StudentProgress/set_progress.html.ep

```diff
@@ -66,30 +66,6 @@
 %
 % # Table description. Only show the problem description if the problems column is shown.
 <div>
-	% if (!$setIsVersioned || $showColumns->{problems}) {
-		<p>
-			<%= maketext(
-				'A period (.) indicates a problem has not been attempted, and a number from 0 to 100 indicates '
-					. 'the grade earned. The number on the second line gives the number of incorrect attempts.'
-			) =%>
-		</p>
-	% }
-	% if ($setIsVersioned) {
-		<p>
-			<%= maketext(
-				q{Click a student's name to see the student's test summary page. }
-					. q{Click a test's version number to see the corresponding test version. }
-					. 'Click a heading to sort the table.'
-			) =%>
-		</p>
-	% } else {
-		<p>
-			<%= maketext(
-				q{Click a student's name to see the student's homework set. Click a heading to sort the table.}
-			) =%>
-		</p>
-	% }
-	%
```

Comment:
I think that you should leave this here (except the part about the period indicating an unattempted problem part).  It doesn't hurt to have the help in the page as well as in the help file.  In fact it is probably better to have it both places.

Reply:
That may be.  Add this in another pull request for consideration.